### PR TITLE
Basic Setup of Disnake

### PR DESCRIPTION
**Pull request by Vinyzu**
- Setup first Basic ReadMe
- Remove Docs from Dev Requirements
- Add Requirements.txt (with Disnake Requirement)
- Setup Basic Disnake Template main.py

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,184 +1,30 @@
-# Python Discord Code Jam Repository Template
```
Comment (WassCodeur) [inline README.md:1]: Hi @Vinyzu , Good work. 

it's LGTM, just that I think we need to improve the README afterwards.

File: main.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import disnake
+from disnake.ext import commands
+
+TOKEN = os.getenv("BOT_TOKEN")
+
+intents = disnake.Intents.default()
+bot = commands.Bot(command_prefix=commands.when_mentioned, intents=intents)
+
+
```
Comment (WassCodeur) [inline main.py:11]: great

**Review by WassCodeur — APPROVED** [review]
Approving...

**Review by WassCodeur — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Vinyzu) [reply README.md:1]: Yes, i would describe the ReadMe as an active WIP. Will push more changes over time and improve it.

**Review by koviubi56 — APPROVED** [review]
it's good for now

**Pull request merged**